src/custom/connectors/index.ts

```diff
@@ -4,7 +4,7 @@ import { WalletConnectConnector } from '@web3-react/walletconnect-connector'
 import { WalletLinkConnector } from '@web3-react/walletlink-connector'
 import { PortisConnector } from '@web3-react/portis-connector'
 
-import { FortmaticConnector } from 'connectors/Fortmatic'
+import { FortmaticConnector, getFortmaticApiKey } from 'custom/connectors/Fortmatic'
```

Comment:
shouldn't need `custom` in the import
```suggestion
import { FortmaticConnector, getFortmaticApiKey } from 'connectors/Fortmatic'
```